api/openapi.yaml.in

```diff
@@ -1177,6 +1177,9 @@ components:
         cleanup_completed_session_ttl:
           type: string
           description: Delete complete sessions that are older than cleanup_completed_session_ttl (in hours). 0h disables cleanup behavior.
+        clear_stage:
```

Comment:
Question: Does openapi automatically generate this from the added options or did you have to manually add these in?

Reply:
This file is the source of truth for our API; I specified these lines and the openapi code generator created the correct schema to match.